packages/dashboard/src/customization/propertiesSections/propertiesPanel/propertiesAndAlarmsTab.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import SpaceBetween from '@cloudscape-design/components/space-between';
+
+import { PropertiesAndAlarmsSettingsConfiguration } from '../propertiesAndAlarmsSettings';
+
+export const PropertiesAndAlarmsSection = () => (
```

Comment:
this component does not seem to have much purpose except adding space... is PropertiesAndAlarmsSettingsConfiguration is used in multiple places ? if not , may be we can add the spaceBetween to the  PropertiesAndAlarmsSettingsConfiguration and not have another component.